src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.PackTool.targets

```diff
@@ -49,7 +49,7 @@ Copyright (c) .NET Foundation. All rights reserved.
         <PackagePath>tools/$(targetframework)/any/%(_GeneratedFiles.RecursiveDir)%(_GeneratedFiles.Filename)%(_GeneratedFiles.Extension)</PackagePath>
       </TfmSpecificPackageFile>
 
-      <TfmSpecificPackageFile Include="@(ResolvedFileToPublish->'$(PublishDir)%(RelativePath)')">
+      <TfmSpecificPackageFile Include="@(ResolvedFileToPublish->'$([MSBuild]::NormalizePath($(PublishDir)))/%(RelativePath)')">
```

Comment:
I think NormalizeDirectory will add trailing slash if needed and then we can avoid that /.

Reply:
https://github.com/Microsoft/msbuild/blob/master/src/Build/Evaluation/IntrinsicFunctions.cs#L409

you are right. Let me just use NormalizeDirectory